libraries/AP_RangeFinder/AP_RangeFinder.h

```diff
@@ -193,9 +193,14 @@ class RangeFinder
         Good
     };
 
+    static constexpr int8_t SIGNAL_QUALITY_MIN = 0;
+    static constexpr int8_t SIGNAL_QUALITY_MAX = 100;
+    static constexpr int8_t SIGNAL_QUALITY_NODATA = -1;
```

Comment:
I think we should rename this definition to SIGNAL_QUALITY_UNKNOWN just so that it is more clear that this is used when we don't have a signal quality (because the sensor doesn't support it) and the case where the rangefinder is not providing distances.

Reply:
Renamed. Thanks.